data/tutorials/lg_01_functional_programming.md

```diff
@@ -11,7 +11,7 @@ date: 2021-08-06T17:16:00-00:00
 
 ## What Is Functional Programming?
 
-At its core, and from its inception, around 1985, OCaml is a [functional
+At its core, and from its origin, around 1985, OCaml is a [functional
```

Comment:
Do you like “origin” better than “inception”? How about “from its debut” ?

At its core, and from its debut around 1985, OCaml is a ….